<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextParser</class>
 <widget class="QDialog" name="TextParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Parser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>Extract</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="extract_btn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="_information">
     <property name="text">
      <string>Text Parser v3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="source"/>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="preview_btn">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="separator_lbl">
     <property name="text">
      <string>Separators:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QCheckBox" name="sep_tab">
     <property name="text">
      <string>\t</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="sep_other_btn">
     <property name="text">
      <string>other:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="sep_other">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="sep_line">
     <property name="text">
      <string>\n</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="sep_space">
     <property name="text">
      <string>space</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="sep_coma">
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>source</tabstop>
  <tabstop>preview_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>extract_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>extract_btn</sender>
   <signal>clicked()</signal>
   <receiver>TextParser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>TextParser</receiver>
   <slot>extract()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preview_btn</sender>
   <signal>clicked()</signal>
   <receiver>TextParser</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sep_other_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>sep_other</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>extract()</slot>
  <slot>preview()</slot>
 </slots>
</ui>
